src/Plugin.php

```diff
@@ -272,18 +276,25 @@ private function saveInstalledPaths()
     protected function getPhpExecCommand()
```

Comment:
FYI: I'd left the code in this function as close to the original as possible on purpose, with only minor CS changes to allow the build to pass.
That way it would be easier to compare the code with the source and update it if/when needed.